test_server.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+cd fishtest
+python setup.py test
+cd ..
```

Comment:
The
    cd ..
at the end of the scripts can be removed

Reply:
@tomtor I agree